repository/src/main/java/org/apache/atlas/discovery/EntityDiscoveryService.java

```diff
@@ -1013,6 +1013,49 @@ public AtlasSearchResult directIndexSearch(SearchParams searchParams) throws Atl
         return ret;
     }
 
+    public int getQueryResponseCount(SearchParams searchParams) throws AtlasBaseException, IOException {
+        IndexSearchParams params = (IndexSearchParams) searchParams;
+        String indexName = getIndexName(params);
+        AtlasIndexQuery indexQuery = graph.elasticsearchQuery(indexName);
+        return indexQuery.getNumVertices(searchParams);
+
+    }
+
+    @Override
+    public DirectIndexQueryResult directIndexSearch(SearchParams searchParams, boolean returnVertices) throws AtlasBaseException {
```

Comment:
returnVertices - this parameter is not being used.

Reply:
USed for function overloading purpose

Reply:
Removed this part alltogether, we can ignore this